app/screens/InfoScreen/SponsorCard.tsx

```diff
@@ -92,15 +122,18 @@ export const SponsorCard = ({
           <Text preset="primaryLabel" style={$sponsorType} tx="infoScreen.bronzeSponsor" />
           <View style={$sponsorBottomTierLogos}>
             {sponsorImages.map(({ uri, sponsor }, index) => (
-              <AutoImage
-                key={index}
-                accessibilityLabel={sponsor}
-                style={[
-                  $sponsorImageBottomTier,
-                  index % 2 === 0 ? $sponsorImageBottomTierStart : $sponsorImageBottomTierEnd,
-                ]}
-                source={{ uri }}
-              />
+              <View key={index} style={$sponsorBottomTierLogoWrapper}>
```

Comment:
I wrapped the image in another view for styling purposes only. I could eliminate the containing wrapper,  by moving those styles to the `sponsorContainer`. However, I would only be eliminating a single view. This seemed simpler, but would love feedback.

Reply:
Do you mind posting screenshots of the rest of the screen too? I'd like to see how the "Bronze" tier displays after this change

Reply:
awesome, thank you! looks good to me, i would keep it that way IMO